content/en/tracing/setup_overview/setup/php.md

```diff
@@ -317,6 +317,32 @@ To remove the PHP tracer:
 
 **Note**: If you are using second level caching in OPcache by setting the parameter `opcache.file_cache`, remove the cache folder.
 
+## Troubleshooting an application crash
+
+While very rare, it can happen that the PHP tracer causes an application to crash. Typically this happens because of a segmentation fault.
+When this happens, the best path ahead is to reach out to support providing a description of the problem and a core dump. Unless the procedure to recreate the application crash is trivial, without a core dump there is very little we can do to help.
```

Comment:
In the unusual event of an application crash caused by the PHP tracer, typically because of a segmentation fault, the best thing to do is obtain a core dump and contact Datadog support.